hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/src/test/java/com/google/cloud/bigtable/hbase2_x/replication/HbaseToCloudBigtableReplicationEndpointTest.java

```diff
@@ -425,4 +411,37 @@ public void testWriteFailureToBigtableDoesNotStallReplication()
         "Value mismatch", TestUtils.getValue(0), CellUtil.cloneValue(actualCells.get(1)));
     Assert.assertEquals(0, actualCells.get(1).getTimestamp());
   }
+
+  @Test
+  public void testMutationReplicationWithWALEntryFilter() throws IOException, InterruptedException {
```

Comment:
Can you add this test to 1.x as well? 

Also, Can you add tests for other filters (scope filter, custom filter etc).

Reply:
Added a scope filter test.